test/app_profiler/backend/vernier_backend_test.rb

```diff
@@ -77,7 +77,7 @@ def teardown
         assert_instance_of(AppProfiler::VernierProfile, profile)
         assert_equal("wowza", profile.id)
         assert_equal("bar", profile.context)
-        assert_equal("spam", profile[:meta][:extrameta])
+        assert_equal("spam", profile.metadata[:extrameta])
```

Comment:
This was already a bit leaky, we should have been accessing it through the metadata helper method.